tests/datasets/test_in_context_learning_datasets.py

```diff
@@ -480,6 +565,48 @@ def test_mc_task_evaluation(device, num_fewshot, dataset_uri, tiny_gpt2_tokenize
     assert in_memory_logger.data['metrics/mc/InContextLearningMultipleChoiceAccuracy'][0][1].item() > 0
 
 
+@pytest.mark.parametrize('dataset_uri', ['triviaqa_small.jsonl'])
+@device('gpu')
+@world_size(1, 2)
+@pytest.mark.parametrize('num_fewshot', [0, 5])
+def test_qa_task_evaluation_opt_tokenizer(device, world_size, num_fewshot, dataset_uri, tmp_path):
+    pytest.importorskip('datasets')
+    in_memory_logger = InMemoryLogger()  # track the logged metrics in the in_memory_logger
+    local_data = os.path.join(os.path.dirname(__file__), 'local_data')
+    dataset_uri = f'{local_data}/{dataset_uri}'
+    tokenizer = AutoTokenizer.from_pretrained('facebook/opt-125m', use_fast=False)
```

Comment:
you shouldn't have to do `use_fast=False` now that we're on the latest transformers release. Maybe you weren't upgrade locally? Or was it failing in CI too?